resources/profiles/Elegoo/machine/Elegoo Neptune 4 Pro (0.2 nozzle).json

```diff
@@ -6,56 +6,30 @@
 	"instantiation": "true",
 	"inherits": "fdm_neptune_4_common",
 	"printer_model": "Elegoo Neptune 4 Pro",
-	"default_print_profile": "0.20mm Standard @Elegoo Neptune4Pro (0.2 nozzle)",
```

Comment:
I haven't enforced the JSON format before.
But since you are formatting them, can you format them with "Prettify JSON" to format them?

Reply:
Will do.